build.sbt

```diff
@@ -13,6 +13,17 @@ lazy val appName = "api-simulator"
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
 lazy val bootstrapVersion = "7.12.0"
 
+ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+
+inThisBuild(
+  List(
+    scalaVersion := "2.12.15",
```

Comment:
I think this should match the version at line 61.

Reply:
yep, fixed cheers. I tried changing both to 2.12.15 and it broke the build so I just aligned them to 2.12.12.
Thanks for spotting!